About.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Buganda Spirit</title>
+</head>
+<body>
+    <div>
+        <div class="top-mobile">
+
+        </div>
+    <div class="topcontact">
+        <ul>
+            <li><img src="https://img.icons8.com/ios-glyphs/18/ffffff/facebook-f.png"/></li>
+            <li><img src="https://img.icons8.com/ios-glyphs/18/ffffff/twitter--v1.png"/></li>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+    <header>
+  
+        <nav class="navbar">
+           <h1> <a href="./index.html"><span>Buganda</span> KingsMen</a></h1>
+            <ul>
+                <li><a href="#" >About</a></li>
+                <li><a href="#" >Program</a></li>
+                <li><a href="#" >Join</a></li>
+                <li><a href="#" >Sponser</a></li>
+                <li><a href="#" >New</a></li>
+                <li><button>CC Campaign</button></li>
+            </ul>
+       
+        </nav>
+</header>
+<div>
+    <div class="headding">
+        <div>.</div>
+      <ul class='Navmenu'>
+        <li class="nav-items"><a class="nav-links" href="./index.html">Home</a> </li>
+        <li class="nav-items"><a class="nav-links" href="./About.html">About</a> </li>
+        <li  class="nav-items"><a class="nav-links"  href="#about">Program</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">Join</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">Sponser</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">New</a></li>
+        <li  class="nav-items"><a class="nav-links"  href="#contact">CC Campaign</a></li>
+      </ul>
+      <div class="hamburger-menu">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+
+    </div>
+</div>
+<section class="intro-sections">
+        <div class="intro-wrapper">
+            <h3 class="intro-wrappers-h3">"The Buganda Pride"</h3>
+            <h1>THE BUGANDA CULTURAL PROSPEROUS <br/>FUTURE FLEX  2022</h1>
+        </div>
+        <p>
+            Buganda is a Bantu kingdom within Uganda. The kingdom of the Baganda people, Buganda is the largest of the traditional kingdoms in present-day East Africa, consisting of Buganda's Central Region, including the Ugandan capital Kampala. The 14 million Baganda (singular Muganda; often referred to simply by the root word and adjective, Ganda) make up the largest Ugandan region, representing approximately 26.6% of Uganda's population.
+        </p>
+        <br>
+        <br>
+        <span>Please contact us via Email for any further questions about the Science Conference 2023</span>
+        <br>
+        <br>
```

Comment:
Kindly, I suggest you use only 1 "BR" tag at a time, to follow good HTML practice